tests/nimble_statconn_gnrc/Makefile

```diff
@@ -7,7 +7,6 @@ USEMODULE += shell_commands
 USEMODULE += ps
 
 # Include GNRC and RPL
-USEMODULE += gnrc_netdev_default
```

Comment:
Here too.

Reply:
For me the binary does not change when I add `nimble_netif`